app/src/main/java/com/devindi/wallpaper/settings/SettingsAdapter.kt

```diff
@@ -0,0 +1,31 @@
+package com.devindi.wallpaper.settings
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import com.devindi.wallpaper.R
+import com.devindi.wallpaper.settings.model.IntField
+import com.devindi.wallpaper.settings.size.SizeViewHolder
+import com.devindi.wallpaper.source.OnItemClickListener
+
+class SettingsAdapter(private val clickListener: OnItemClickListener) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+
+    var items: List<IntField> = emptyList()
+        set(value) {
+            field = value
+            notifyDataSetChanged()
+        }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerView.ViewHolder {
+        val view = LayoutInflater.from(parent.context).inflate(R.layout.settings_simple_item, parent, false)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/devindi/mapwall/blob/264baa37465d6b2f1e3fbd000c3445362f879c9d/app/src/main/java/com/devindi/wallpaper/settings/SettingsAdapter.kt#L20"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>